### DISCUSSION TITLE
How to set vertical ruler color using `options.lua`

### DISCUSSION THREAD

**Discussion by Fl4m3Ph03n1x** [LazyVim/LazyVim#2550] [Q&A] [answered]
In my `config/options.lua` I have the set  a vertical ruler to help me code:

```
local opt = vim.opt
opt.colorcolumn = "120"
```

However now I want to change the color of the ruler. I know the command to use is:
`:highlight ColorColumn guibg=lightblue`

But I don't know how to set it as an option. 

How can I do this?

**Comment by dpetka2001** [comment, marked as answer]
Try setting it with an auto-command like so
```lua
vim.api.nvim_create_autocmd("BufReadPost", {
  group = vim.api.nvim_create_augroup("ColorColumn", { clear = true }),
  desc = "Change ColorColumn highlight",
  callback = function()
    vim.api.nvim_set_hl(0, "ColorColumn", { bg = "red" })
  end,
})
```
Put that in your `~/.config/nvim/lua/config/autocmds.lua` and change the `bg` value to whatever color you'd like.
